src/loaders/OMETiffLoader.js

```diff
@@ -159,10 +159,10 @@ export default class OMETiffLoader {
         }
       }
       image = await tiff.getImage(pyramidIndex);
-      return this._getChannel({ image, x, y, z });
+      return this._getChannel({ image, x, y, z, signal });
     });
-
     const tiles = await Promise.all(tileRequests);
+    if (signal.aborted) return null;
```

Comment:
Would it make sense to check the signal.aborted before await-ing?

```suggestion
    if (signal.aborted) return null;
    const tiles = await Promise.all(tileRequests);
```

Reply:
I don't think so, because if the signal aborts, geotiff still returns so we need to check after returning.